### ISSUE TITLE
Add option to change DEFAULT_TIMEOUT globally

### ISSUE THREAD

**Issue by douglasjunior** [callstack/react-native-testing-library#953]
## Describe the Feature

Add option to change default configuration (like `DEFAULT_TIMEOUT`) globally, like in "@testing-library/dom" https://testing-library.com/docs/dom-testing-library/api-configuration/

## Possible Implementations

```js
import {configure} from '@testing-library/react-native'

configure({
  asyncUtilTimeout: 5000,
  // etc...
})
```

## Related Issues

https://github.com/testing-library/react-hooks-testing-library/issues/812

**Comment by mdjastrzebski** [comment]
Closing as implemented in in #1141.

**Comment by douglasjunior** [comment]
Wow, amazing! 

Thank you, @mdjastrzebski!
